RecetarioExpo/src/components/SearchText.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { StyleSheet, View, TextInput } from 'react-native';
+import { Icon } from 'react-native-elements';
+
+
+export default function SearchText() {
+  return (
+    <View style={styles.container}> 
+        <View style={styles.content}>
+            <Icon 
+                color= '#fff'
+                name='search'
+            />
+            <TextInput 
+                style={styles.text} 
+                placeholder="What do you want to eat?"
+                placeholderTextColor='#ffffffbc'
+            >
+            </TextInput>
+            <Icon 
+                color= '#fff'
+                name='mic'
+            />
+        </View>
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    alignItems: 'center',
+    flex: 0,
+    flexDirection: 'row',
+    justifyContent: 'center',
+    width: '100%',
+  },
+  content:{
+    backgroundColor: '#2c2c2c',
+    borderRadius: 10,
+    flex: 0,
+    flexDirection: 'row',
+    justifyContent: 'space-around',
+    marginTop: 5,
+    marginBottom: 5,
+    padding: 5,
+    width: '90%',
+  },
+  text:{
+    color: '#FFF',
+    width: '80%',
+  }
+});
```

Comment:
Styles can be placed in separate files, and imported into this file.

As a rule of the Style Guide, always leave a blank line at the end of each file.